docs/content/en/docs/troubleshooting/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Troubleshooting"
+weight: 8
+icon: "faq"
+---
```

Comment:
Can you add a little description for it to appear here https://deploy-preview-1664--tetragon.netlify.app/docs/

Reply:
Done, thanks!